internal/testrunner/runners/system/test_config.go

```diff
@@ -19,10 +20,11 @@ import (
 	"github.com/elastic/elastic-package/internal/testrunner/runners/system/servicedeployer"
 )
 
-const configFileName = "config.yml"
+var systemTestConfigFilePattern = regexp.MustCompile(`^test-([a-z0-9_.-]+)-config.yml$`)
 
 type testConfig struct {
 	Input      string                       `config:"input"`
+	Service    string                       `config:"service"`
```

Comment:
@mtojek This is more of a note for us for the future: as we add more service deployers we will need to make sure this field continues to work as expected with the new service deployers as well (not just the Docker compose one that we have today).